application/dash_application/plot_builder.py

```diff
@@ -149,6 +159,18 @@ def generate(title_text, sorted_data, x_fn, y_fn, axis_text, colourby, shapeby,
             line={"width": 1, "color": "black", "dash": "dash"},
             name="Cutoff"
         ))
+    if not highlight_df.empty:
+        traces.append(go.Scattergl( # Draw highlighted items on top
+            x=x_fn(highlight_df),
+            y=y_fn(highlight_df),
+            name="Highlighted Samples",
```

Comment:
Can the name of the sample be displayed instead?

Reply:
Currently all the markers are drawn in one trace, which only allows for 1 legend entry. I'll look into looping over them and whether that's too slow for high # of highlighted samples

Reply:
So it seems as though when looping over samples one at a time (rather than in groups of them) it doesn't understand what to do when given a Sample name for the x co-ordinate. I think this feature might need further discussion.